### DISCUSSION TITLE
Kernel Muere en el input

### DISCUSSION THREAD

**Discussion by ArmandoRuiz13** [LaunchX-InnovaccionVirtual/CursoIntroPython#84] [Q&A] [answered]
[
![3](https://user-images.githubusercontent.com/99087774/153475803-a4831720-e9cf-4952-a5dc-6a49df73df62.PNG)
](url)

Cuando trato de ejecutar un input muere el kernel, pasa en la extension .ipynb sera cosa de la version?

**Comment by doguedogue** [comment]
Prueba en la terminal si te hace lo mismo, normalmente recoges en una variable lo que pides
$ python3
>>>text = input("....... ..... .... ")

NOTA: No olvides marcar como contestada si así lo consideras!

**Reply by ArmandoRuiz13** [reply]
No funciona, sigue muriendo.

**Reply by ArmandoRuiz13** [reply]
Funciona haciendolo fuera del .ipynb

**Comment by JoseEGJasso** [comment, marked as answer]
el kernel se muere instantaneamente después de que lo ejecutas? no te aparece un cuadro dialogo como este?
 
![image](https://user-images.githubusercontent.com/41630679/153484155-fa5294ba-a4ab-4f8e-8caf-aeb59811aa71.png)

**Reply by ArmandoRuiz13** [reply]
Muchas gracias, parece que solo hice el oso. No sabes si hay una forma de que aparezcan en otro lado para dar las respuestas? Como un cmd sin cells

**Reply by JoseEGJasso** [reply]
Jaja me paso lo mismo, se camuflajea muy gacho con el fondo. La verdad no sé si se pueda sin utilizar alguna biblioteca que lo genere aparte en una ventana

**Reply by ADIGARCIA** [reply]
Igual muchas gracias @JoseEGJasso , si aparece la barra arribita para colocar la respuesta al input :)
